jsapp/js/universalTable/universalTable.component.tsx

```diff
@@ -129,11 +134,24 @@ export default function UniversalTable<DataItem>(
   }
 
   function getCommonClassNames(column: Column<DataItem>) {
+    const isPinned = column.getIsPinned();
     return cx({
-      [styles.isPinned]: Boolean(column.getIsPinned()),
+      [styles.isPinnedLeft]: isPinned === 'left',
+      [styles.isPinnedRight]: isPinned === 'right',
+      [styles.isLastLeftPinnedColumn]: isPinned === 'left' && column.getIsLastColumn('left'),
+      [styles.isFirstRightPinnedColumn]: isPinned === 'right' && column.getIsFirstColumn('right'),
     });
   }
 
+  function getCommonColumnStyles(column: Column<DataItem>): CSSProperties {
+    const isPinned = column.getIsPinned();
+    return {
+      left: isPinned === 'left' ? `${column.getStart('left')}px` : undefined,
+      right: isPinned === 'right' ? `${column.getAfter('right')}px` : undefined,
+      width: `${column.getSize()}px`,
+    };
+  }
```

Comment:
There's a gap between the pinned columns
![image](https://github.com/user-attachments/assets/c5031982-ed77-4192-8419-8aa7679186d4)

Reply:
Thanks, fixed 👍